tracetools_launch/tracetools_launch/action.py

```diff
@@ -221,11 +226,11 @@ def context_fields(
         return self._context_fields
 
     @property
-    def subbuffer_size_ust(self) -> int:
+    def subbuffer_size_ust(self) -> Union[int, List[SomeSubstitutionsType]]:
```

Comment:
I think this should be:

```suggestion
    def subbuffer_size_ust(self) -> Union[int, List[Substitution]]:
```

right? Since `normalize_typed_substitution()` returns `List[Substitution]` or `int`. `SomeSubstitutionsType` is pre-normalization.

Reply:
ah, yes! But should I change it to `NormalizedTypeValue` instead, like you suggested for `append_trace`?

Reply:
oops, yes!